Tests/FoundationTests/DateExtensionsTests.swift

```diff
@@ -556,14 +556,16 @@ final class DateExtensionsTests: XCTestCase {
 		date.add(.month, value: 1)
 		XCTAssertEqual(date.month, 10)
 		
-		date = Date()
-		date.add(.year, value: -1)
-		XCTAssertEqual(date.year, 2016)
-		date.add(.year, value: 0)
-		XCTAssertEqual(date.year, 2016)
+        var date1 = Date(integerLiteral: 2018_01_01)
```

Comment:
I'd prefer to be explicit here and use Apple's initializer

```swift
let date = Date(timeIntervalSince1970: 1514764800)
```

Reply:
Done. Fixed to default initializer